community/cypher/cypher/src/main/java/org/neo4j/cypher/internal/codegen/ParameterConverter.java

```diff
@@ -239,8 +238,8 @@ else if ( obj instanceof Path )
             {
                 return new Iterator<PropertyContainer>()
                 {
-                    Iterator<? extends PropertyContainer> current = nodes().iterator();
-                    Iterator<? extends PropertyContainer> next = relationships().iterator();
+                    private Iterator<? extends PropertyContainer> current = nodes().iterator();
+                    private Iterator<? extends PropertyContainer> next = relationships().iterator();
```

Comment:
This is just more typing. Not that it matters much, but I fail to see the point.

Reply:
"Failing to explicitly declare the visibility of a member variable could result it in having a visibility you don't expect, and potentially leave it open to unexpected modification by other classes."

I do not need it, but it might be nice for people less experienced with java defaults? Say the word and I kill it.

Reply:
Since it is an anonymous class, the type itself is not visible, and thus the fields are not visible.